src/core/infrastructure/error.d.ts

```diff
@@ -0,0 +1,7 @@
+export declare class AppError{
+
+	constructor(name: String, message: String);
+
+	name: String;
```

Comment:
I guess no need to declare name and message AppError used only as exception source